### DISCUSSION TITLE
Identity spoofing

### DISCUSSION THREAD

**Discussion by EP-GF** [grafana/grafana#52076] [General]
Hello Grafana team,

I would like to report a possible security vulnerability for discussion.  The version I'm using is 7.1.5 the free download version on RHEL.

![image](https://user-images.githubusercontent.com/109121236/178394285-05cb008f-282a-45b0-a159-593b8eeaa05f.png)

Little John is a valid user in Grafana.  Little John edits his profile as Big Tom and hits Save, and the profile is updated.

![image](https://user-images.githubusercontent.com/109121236/178394507-1a0bf9f1-7611-444f-9ec6-ecba92f371a4.png)

Little John then signs out.

In Admin's the Users list, bigtom shows up instead of littlejohn.

![image](https://user-images.githubusercontent.com/109121236/178394964-bf9010c7-7c14-451e-92a7-3192a7fe32bf.png)
![image](https://user-images.githubusercontent.com/109121236/178395188-b4827d94-183a-44a8-a26c-60ffbad83d4d.png)

The Admin may think Big Tom is really the intended big tom and possibly assign Big Tom elevated permissions.
Ta-da! Little John can then login as usual with elevated permissions.

Is there anything I can do to prevent users from editing their profile?  At the very least I think the username should not be editable.

**Comment by EP-GF** [comment]
Found Disable Profile option since v8.5.0.  Thanks and sorry for the trouble.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
